showcase/src/app/demo/pretty-print/pretty-print.component.html

```diff
@@ -0,0 +1,22 @@
+<example-viewer *ngFor="let fileType of fileTypes; trackBy: trackFileTypes"
+				[extensions]="['HTML', 'SCSS', 'TS']"
+				[filesPath]="fileType.path"
+				[title]="fileType.title">
+	<mat-tab-group>
+		<mat-tab label="Original Code">
+			<div class="code-block">{{ fileType.rawData }}</div>
+		</mat-tab>
+		<mat-tab label="Pretty Printed">
+			<div class="code-block">
+				<stark-pretty-print [data]="fileType.rawData"
+									[format]="fileType.format"
+									[enableHighlighting]="false"></stark-pretty-print>
+			</div>
+		</mat-tab>
+		<mat-tab label="Pretty Printed & Highlighted">
+			<stark-pretty-print [data]="fileType.rawData"
+								[format]="fileType.format"
+								[enableHighlighting]="true"></stark-pretty-print>
```

Comment:
Could you format these 2 lines?  You can format them automatically with IntelliJ (in the meantime Prettier adds support for HTML).